openqa-investigate

```diff
@@ -34,6 +34,7 @@ clone() {
     name_suffix=${3+":$3"}
     refspec=${4+$4}
     local clone_job_data
+    vars_json=$(fetch-vars-json "$origin")
```

Comment:
I don't think you should pull this out of the `if refspec` as that would mean that every "retry" job would now also need to parse the origin's vars.json file which should be avoided